get_started/simple_pipeline.md

```diff
@@ -0,0 +1,247 @@
+	    output_caps: %Membrane.Caps.Audio.Raw{
+		  format: :s16le,
+		  sample_rate: 48000,
+		  channels: 2
+		}
+	  },
+	  portaudio: Membrane.PortAudio.Sink
+    }
+
+	links = [
+	  link(:file)
+	  |> to(:decoder)
+	  |> to(:converter)
+	  |> to(:portaudio)
+	]
+
+	spec = %ParentSpec{children: children, links: links}
+
+	{{:ok, spec: spec}, %{}}
+  end
+end
+```
+That might not look too simple for now but don't worry, there'll be a lot of new things you're going to meet, and we'll be introducing you to some of them, or giving hints how and where to learn more.
+
+> #### Elixir
+> Membrane is written in Elixir. It's an awesome programming language of functional paradigm with great fault tolerance and process management, which made it a best choice for Membrane.
+> If you're not familiar with it, you can use [this cheatsheet](https://devhints.io/elixir) for quick look-up.
+> We encourage you also to take a [deep look into Elixir](https://elixir-lang.org/getting-started/introduction.html) and learn how to use it to take advantage of all it's awesomeness. We believe you fall in love with Elixir too!
+>
+> To play and work with membrane you'll need to have Elixir environment installed on your system. Instructions how to do it depending to your operating system you'll find [here](https://elixir-lang.org/install.html).
+
+
+The code above is one of the simplest examples of Membrane usage. It plays mp3 file through your device's `portaudio`. Let's make it work.
+
+### Prerequisites
+
+First we need to get all libraries that Membrane needs to operate in our case. You can read about them more if you'd like, but for now we'll just jump to installation:
+
+##### Ubuntu
+```bash
+$ apt install clang-format portaudio19-dev ffmpeg libavutil-dev libswresample-dev libmad0-dev
+```
+##### Mac
+```bash
+$ brew install clang-format portaudio ffmpeg libmad pkg-config
+```
+
+### Creating a Project
+
+While installing Elixir you'll get a bunch of useful tools. One of them is [Mix](https://hexdocs.pm/mix/Mix.html). As you can read in its documentation preface:
+> Mix is a build tool that provides tasks for creating, compiling, and testing Elixir projects, managing its dependencies, and more.
+
+Lets use it to create a project for our first Membrane adventure:
+
+```bash
+$ mix new hello --module Hello
+```
+
+Mix generator will create some files for us. Lets take a closer look into two of them:
+
+**mix.exs** - It's an entry file for our mix project, a place where we can configure it, and set-up our dependencies. We'll do it by adding them into `deps` function:
+
+```elixir
+defp deps do
+  [
+	{:membrane_core, "~> 0.7.0"},
+	{:membrane_file_plugin, "~> 0.6.0"},
+	{:membrane_portaudio_plugin, "~> 0.7.0"},
+	{:membrane_ffmpeg_swresample_plugin, "~> 0.7.1"},
+	{:membrane_mp3_mad_plugin, "~> 0.7.0"}
+  ]
+end
+```
+Next we can fetch them by running
+```bash
+$ mix deps.get
+```
+
+**lib/hello.ex** - Here we'll be creating our pipeline.
+
+### Our first Pipeline
+
```

Comment:
I would add a quick intro sentence stating that pipelines are one of the standard elements used by Membrane and that they function to take in some data, push it through a series of elements that transform the data and output it somewhere.